client/my-sites/plugins/plugin-details-CTA/jetpack-search-cta.jsx

```diff
@@ -0,0 +1,41 @@
+import { WPCOM_FEATURES_INSTANT_SEARCH } from '@automattic/calypso-products';
+import { Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-transfer';
+import siteHasFeature from 'calypso/state/selectors/site-has-feature';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import getSiteAdminUrl from 'calypso/state/sites/selectors/get-site-admin-url';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import PreinstalledCTA from './preinstalled-cta';
+
+export default function PluginDetailsCTAJetpackSearch( { pluginName } ) {
+	const siteId = useSelector( getSelectedSiteId );
+	const siteSlug = useSelector( getSelectedSiteSlug );
+	const isAtomic = useSelector( ( state ) => isSiteAutomatedTransfer( state, siteId ) );
+	const isJetpack = useSelector( ( state ) => isJetpackSite( state, siteId ) );
+	const isJetpackSelfHosted = siteId && isJetpack && ! isAtomic;
+	const hasInstantSearch = useSelector( ( state ) =>
+		siteHasFeature( state, siteId, WPCOM_FEATURES_INSTANT_SEARCH )
+	);
+	const wpAdminUrl = useSelector( ( state ) =>
+		getSiteAdminUrl( state, siteId, 'admin.php?page=jetpack-search' )
+	);
+
+	const translate = useTranslate();
+
+	const buttonLabel = hasInstantSearch
+		? translate( 'Manage plugin' )
+		: translate( 'Manage plugin (upgrades available)' );
+
+	const buttonHref = isAtomic ? wpAdminUrl : `/settings/performance/${ siteSlug }`;
```

Comment:
Is `/jetpack-search/${ siteSlug }` for Simple sites deprecated?

Reply:
Maybe have someone from @Automattic/jetpack-search take a look.

One thought, can we make the switch so that we are selling jetpack_search plans here rather than wpcom_search plans? I think that was a part of the reason for needing that route.